Extensions/one_click_postage.js

```diff
@@ -1435,12 +1433,8 @@ XKit.extensions.one_click_postage = new Object({
 					$(m_button).removeClass("xkit-one-click-reblog-working");
 				},
 				onload: function(response) {
-					// We are done!
-					XKit.interface.kitty.set(response.getResponseHeader("X-Tumblr-Kittens"));
-
-					var mdata;
```

Comment:
I'd prefer to keep the declaration of mdata here since it feels better even though `var` scoping is technically just Like That